lib/src/bindings_player_ffi.dart

```diff
@@ -451,6 +477,36 @@ class FlutterSoLoudFfi {
   );
   late final _setLooping = _setLoopingPtr.asFunction<void Function(int, int)>();
 
+  /// Get sound loop point value.
+  ///
+  /// [handle]
+  /// Returns the time in seconds.
+  double getLoopPoint(int handle) {
+    return _getLoopPoint(handle);
+  }
+
+  late final _getLoopPointPtr =
+      _lookup<ffi.NativeFunction<ffi.Double Function(ffi.UnsignedInt)>>(
+          'getLoopPoint');
+  late final _getLoopPoint =
+      _getLoopPointPtr.asFunction<double Function(int)>();
+
+  /// Set sound loop point value.
+  ///
+  /// [handle]
+  /// [time] in seconds.
+  void setLoopPoint(int handle, double time) {
```

Comment:
This should take a `SoundHandle handle`, for consistency.

Reply:
right